pod/perlvar.pod

```diff
@@ -1120,6 +1120,26 @@ information on this form and its uses.
 
 This variable was added in 5.25.7
 
+If you need access to this functionality in older Perl's you can use this
+function immediately after your regexp.
+
+    sub get_captures {
+        my $str      = $&;
```

Comment:
At least in older Perls, the use of `$&` slows down all regex matches everywhere in the program.